### DISCUSSION TITLE
Creating Liouvillian operator with non local dissipation

### DISCUSSION THREAD

**Discussion by molendziak** [netket/netket#1767] [Q&A]
Hello!
Is it possible to construct a Liouvillian operator such that we could include two body non local dissipation, so the equation would look like this if the sum would be over the first nearest neighbours:

$$
\mathcal{L} = -i H_{nh} \rho + i\rho H_{nh}^\dagger + \sum_j L_j \rho L_{j+1}^\dagger
$$

with $H_{nh} = H - \sum_j \frac{i}{2} L^{\dagger}_{j+1}L_j$.  Can this be done through modyfing nk.operator.LocalLiouvillian or would it be necessary to create a custom operator?

**Comment by PhilipVinc** [comment]
You can simply add the two body operator as a dissipator.

I don't see why it should not work.
